tests/components/brother/test_config_flow.py

```diff
@@ -94,10 +94,11 @@ async def test_invalid_hostname(hass: HomeAssistant) -> None:
     assert result["errors"] == {CONF_HOST: "wrong_host"}
 
 
-async def test_connection_error(hass: HomeAssistant) -> None:
+@pytest.mark.parametrize("exc", [ConnectionError(), TimeoutError()])
```

Comment:
```suggestion
@pytest.mark.parametrize("exc", [ConnectionError, TimeoutError])
```
I don't think you need those ()

Reply:
Yup, you're right